engine/kafka/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaConfig.scala

```diff
@@ -6,7 +6,13 @@ import pl.touk.nussknacker.engine.api.process.ProcessObjectDependencies
 case class KafkaConfig(kafkaAddress: String,
                        kafkaProperties: Option[Map[String, String]],
                        kafkaEspProperties: Option[Map[String, String]],
-                       consumerGroupNamingStrategy: Option[ConsumerGroupNamingStrategy.Value] = None)
+                       consumerGroupNamingStrategy: Option[ConsumerGroupNamingStrategy.Value] = None) {
+
+  def forceLatestRead: Option[Boolean] = kafkaEspProperties.flatMap(_.get("forceLatestRead")).map(_.toBoolean)
+
+  def defaultMaxOutOfOrdernessMillis: Option[Long]
+    = kafkaEspProperties.flatMap(_.get("defaultMaxOutOfOrdernessMillis")).map(_.toLong)
```

Comment:
How this default can be overridden? In future you want to add maxOutOfOrderness on source level?

Reply:
Ignore this question.